tasks/create_fs.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: manage_lvm | unmounting filesystem(s)
+  mount:
+    path: "{{ lv.mntp }}"
+    src: "/dev/{{ vg.vgname }}/{{ lv.lvname }}"
+    fstype: "{{ lv.filesystem | default(omit) }}"
+    state: "absent"
+  become: true
+  loop: "{{ vg.lvnames }}"
+  loop_control:
+    loop_var: lv
+  when: >
+    (lv is defined and
+    lv != 'None') and
+    (lv.create is defined and
+    not lv.create and
+    lv.filesystem != "swap")
```

Comment:
If you pull this task out of this file, you could add the following condition to the include, and remove it from the tasks in this file:

```
when: vg.create is defined and vg.create
```

Same applies to create_lv.yml

Reply:
I just took the original guards: I don't even know why we should check that!
To me, lv.create check should be sufficient to check the fs creation?

Reply:
I think the guards are right - we may want to delete a VG by setting vg.create = false. Same for an LV. We also may want to create a VG and LV, but not a file system, or create a file system but not mount it.